en/docs/guides/user-mgt/tenant-wise-config.md

```diff
@@ -0,0 +1,132 @@
+# Tenant-Wise Email Sender Configuration
+
+When handling notifications such as the ones given below, the email-sender configuration needs to be changed in `<IS-HOME>/repository/conf/deployment.toml`. 
```

Comment:
there is already a page for email configurations called "configure-email-sending.md" in the fragments folder. we can directly link to that page or include it here

Reply:
That fragment has way too much information. Since we only needed the toml configuration to show the difference I've added just that part.

Reply:
adding just the toml configuration doesn't give a description about each config and expected values...WDYT about adding the configure-email-sending.md page to a nav and linking to it? its a page that we will need to refer in several places I think

Reply:
Yeah, there might be other pages which might need this as a link too. Let's discuss and place it in an appropriate place.